app/models/profile.rb

```diff
@@ -49,6 +49,10 @@ def activate
 
   def activate_after_passing_review
     update!(fraud_review_pending: false, fraud_rejection: false)
+    irs_attempts_api_tracker&.fraud_review_adjudicated(
+      decision: 'pass',
+      fraud_fingerprint: Digest::SHA1.hexdigest(user.uuid),
```

Comment:
This name is super bikeshed-able.

The idea is that we're passing in what's meant to be an opaque identifier, allowing them to link the TMX check to the pass/reject action and thus determine what user it refers to, so I wanted to give this a fairly generic name to avoid it being relied on for anything.